index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <nav class="header">
+            <a class="logo" href="#">My Logo</a>
+            <img src="images/Union.png" alt="">
+        </nav>
```

Comment:
- [ ] Please wrap the `<nav>` in a `<header>` element. I will suggest that your work looks like this.
```html
<header>
  <a class="logo" href="#">My Logo</a>
  <nav class="header">
    <img src="images/Union.png" alt="">
  </nav>
</header>```